test/e2e.nut.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { expect } from 'chai';
+import { TestSession, execCmd } from '@salesforce/cli-plugins-testkit';
+
+const projectPath = 'testProject_hubAndOrg';
+let session: TestSession;
+
+describe('verifies all commands run successfully ', () => {
+  before(() => {
+    session = TestSession.create({
+      project: {
+        destinationDir: projectPath,
+        sourceDir: 'test/df17AppBuilding',
+      },
+      // create org and push source to get a permset
+      setupCommands: ['sfdx force:org:create -d 1 -s -f config/project-scratch-def.json', 'sfdx force:source:push'],
+    });
+  });
+
+  it('user display', () => {
+    const output = execCmd('force:user:display --json', { ensureExitCode: 0 });
+    expect(output.jsonOutput)
+      .to.have.property('result')
+      .with.keys(['username', 'accessToken', 'id', 'orgId', 'profileName', 'loginUrl', 'instanceUrl']);
+    const result = (output.jsonOutput as Record<string, unknown>).result as Record<string, string>;
+    expect(result.orgId).to.have.length(18);
+    expect(result.id).to.have.length(18);
+    expect(result.accessToken.startsWith(result.orgId.substr(0, 15))).to.be.true;
+  });
+
+  it('assigns a permset to the default user', () => {
+    execCmd('force:user:permset:assign -n VolunteeringApp --json', { ensureExitCode: 0 });
```

Comment:
I like the `with.keys` check. Can we add those to these tests too? It gives us some regression testing around the json output, which is the biggest thing the previous integration test caught.

Reply:
that's the kind of feedback I wanted.  I'll do more of those.

Reply:
added